src/index.js

```diff
@@ -0,0 +1,62 @@
+import './index.css';
+import getProducts from './modules/getProducts.js';
+import itemPopup from './modules/itemPopup.js';
+import closePopup from './modules/closePopUp.js';
+import getResFormData from './modules/getResFormData.js';
+
+import TheLikes from './modules/showLikes.js';
+
+import productCount from './modules/productCount.js';
+
+const store = document.querySelector('.counter');
+
+const logo = document.querySelector('.logo');
+logo.innerHTML = '<div class="oke">O K E</div>';
+
+// Items Interface
+const itemsContainer = async () => {
+  const likes = await new TheLikes().likesStoredData;
+  const theProducts = await getProducts();
+
+  const productsList = document.getElementById('products-list');
+  productsList.innerHTML = '';
+  const getAllProducts = async () => {
+    for (let i = 0; i < theProducts.length; i += 1) {
+      const itemId = i + 1;
+      productsList.insertAdjacentHTML(
+        'beforeend',
+        `
+        <li class="store-items">
+        <img class="store-items-img" src="${theProducts[i].image}"/>
+        <div class="title-like"><h2 class="itemid">Item ${itemId} </h2>
+        <i id="${i}" class="fa fa-heart likebtn" aria-hidden="true"></i>
+        <p class="likes-count">${likes[i].likes}</p>
+        </div>
+        <button id ="comment-btn" class="comment-btn">Comments</button>
```

Comment:
:red_circle: [REQUIRED] I've reviewed the project requirements and noticed that there should be a total of three interfaces in our application:

1. Home Page
2. Comments Popup
3. Reservations Popup


I appreciate the work that has been done so far by the team. However, I've noticed that the Comments Popup interface has not been implemented yet. It's possible that the project requirements might not have been entirely clear, and I understand that such situations can happen.

To move forward, I suggest that you prioritize the implementation of the Comments Popup as soon as possible to align with the project requirements. If there are any questions or uncertainties regarding the requirements or the implementation, please feel free to reach out for clarification.

Reply:
Clearly, you are right. We got it but had a little problem with the availability and code errors.. 😄

Reply:
I am forced to ask for another review because I have no body now to work with to fix what is left. I apply for review as 2 partners, for homepage and reservation.

Reply:
Everything got updated about 10mins ago
Everything works find now.